src/Microsoft.Tye.Core/Pipeline.cs

```diff
@@ -38,9 +38,9 @@ public static void Configure<TApplication>(Action<TApplication> callback)
             foreach (var (type, delegates) in callbacks)
             {
                 var method = pipeline.GetType().GetMethod("Register")!.MakeGenericMethod(type);
-                for (var i = 0; i < delegates.Count; i++)
+                foreach (var @delegate in delegates)
```

Comment:
nit: `@delegate` to `del`

Reply:
Why?

Reply:
Personally not a fan of using `@` variables.

Reply:
Delegate is much cleaner then del. Del could mean to much